AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Dependency/AWSPinpointAnalyticsClientBehavior.swift

```diff
@@ -7,6 +7,7 @@
 
 import AWSPinpoint
 import Foundation
+import StoreKit
```

Comment:
Is StoreKit needed here?

Reply:
For `SKPaymentTransaction` and `SKProduct`